pkg/minikube/machine/cache_images.go

```diff
@@ -106,24 +108,26 @@ func LoadImages(cc *config.MachineConfig, runner command.Runner, images []string
 		return errors.Wrap(err, "runtime")
 	}
 
+	cli, err := client.NewEnvClient() // docker cli for image id
+
```

Comment:
This should check for err before starting the run loop.

Also, I don't understand the comment. I'd probably just remove it.